evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -280,6 +281,19 @@ class DummyEstimator(Estimator):
     expected_parameters = {'Dummy!': {'bar': 'baz'}}
     assert pipeline.parameters == expected_parameters
 
+    X, y = X_y_binary
+    automl = AutoMLSearch(problem_type='binary', objective="Log Loss Binary", max_pipelines=5)
+    automl.search(X, y)
```

Comment:
@christopherbunn why use automl in this unit test? You can just create a pipeline class to use:
```python
class TestPipeline(BinaryClassificationPipeline):
    component_graph = [Imputer, OneHotEncoder, LogisticRegressionClassifier]

pipeline = TestPipeline(parameters={})
...
```
Or, use one already defined in our test fixtures, like [this one](https://github.com/alteryx/evalml/blob/main/evalml/tests/conftest.py#L233).

Reply:
Hmm, I think the intent was to originally compare fitted pipelines, but then I guess that gets fitted separately anyways. I'll update this.